sysml/src/examples/Geometry Examples/CarWithEnvelopingShape.sysml

```diff
@@ -0,0 +1,29 @@
+package CarWithEnvelopingShape {
```

Comment:
These depend on the removed files.  I assume the latest updates are on 507-1.  Could either delete them, or I could replace with examples, if needed.

Reply:
We could add this one as a basic CSG example to sysml/src/examples/Geometry Examples

Reply:
Added this to the Geometry examples as CarWithShapeAndCSG.  Pushed.